README.md

```diff
@@ -1,6 +1,6 @@
 # AutSPACEs
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-36-orange.svg?style=flat-square)](#contributors)
+[![All Contributors](https://img.shields.io/badge/all_contributors-38-orange.svg?style=flat-square)](#contributors-)
```

Comment:
```suggestion
[![All Contributors](https://img.shields.io/badge/all_contributors-38-orange.svg?style=flat-square)](#contributors)
```
It looks like this gained an accidental extra `-` at the end... at least this seems to break the forward link for me (but maybe it's there for a reason that I'm not appreciating?).

Reply:
Great catch! I've fixed it! And it seems that the CLI tool for `all-contributors` adds this, which is weird. I'll check if that's a reported bug!